contracts/vaults/TreasuryVault/RibbonTreasuryVaultWithAirSwap.sol

```diff
@@ -0,0 +1,1165 @@
+    address public immutable WETH;
+
+    /// @notice USDC 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48
+    address public immutable USDC;
+
+    // @notice AirSwap 0x4572f2554421Bd64Bef1c22c8a81840E8D496BeA
+    address public immutable AIRSWAP_CONTRACT;
+
+    /// @notice 15 minute timelock between commitAndClose and rollToNexOption.
+    uint256 public constant DELAY = 0;
+
+    // Number of weeks per year = 52.142857 weeks * FEE_MULTIPLIER = 52142857
+    // Dividing by weeks per year requires doing num.mul(FEE_MULTIPLIER).div(WEEKS_PER_YEAR)
+    uint256 private constant WEEKS_PER_YEAR = 52142857;
+
+    // GAMMA_CONTROLLER is the top-level contract in Gamma protocol
+    // which allows users to perform multiple actions on their vaults
+    // and positions https://github.com/opynfinance/GammaProtocol/blob/master/contracts/core/Controller.sol
+    address public immutable GAMMA_CONTROLLER;
+
+    // MARGIN_POOL is Gamma protocol's collateral pool.
+    // Needed to approve collateral.safeTransferFrom for minting otokens.
+    // https://github.com/opynfinance/GammaProtocol/blob/master/contracts/core/MarginPool.sol
+    address public immutable MARGIN_POOL;
+
+    /// OTOKEN_FACTORY is the factory contract used to spawn otokens. Used to lookup otokens.
+    address public immutable OTOKEN_FACTORY;
+
+    // The minimum amount above which premium distribution will occur during commitAndClose
+    uint256 private constant MIN_DUST_AMOUNT = 10000000;
+
+    /************************************************
+     *  EVENTS
+     ***********************************************/
+
+    event Deposit(address indexed account, uint256 amount, uint256 round);
+
+    event InitiateWithdraw(
+        address indexed account,
+        uint256 shares,
+        uint256 round
+    );
+
+    event Redeem(address indexed account, uint256 share, uint256 round);
+
+    event ManagementFeeSet(uint256 managementFee, uint256 newManagementFee);
+
+    event PerformanceFeeSet(uint256 performanceFee, uint256 newPerformanceFee);
+
+    event CapSet(uint256 oldCap, uint256 newCap);
+
+    event Withdraw(address indexed account, uint256 amount, uint256 shares);
+
+    event CollectManagementFee(
+        uint256 managementFee,
+        uint256 round,
+        address indexed feeRecipient
+    );
+
+    event CollectPerformanceFee(
+        uint256 performanceFee,
+        uint256 round,
+        address indexed feeRecipient
+    );
+
+    event DistributePremium(
+        uint256 amount,
+        uint256[] amounts,
+        address[] recipients,
+        uint256 round
+    );
+
+    event OpenShort(
+        address indexed options,
+        uint256 depositAmount,
+        address indexed manager
+    );
+
+    event CloseShort(
+        address indexed options,
+        uint256 withdrawAmount,
+        address indexed manager
+    );
+
+    event NewOptionStrikeSelected(uint256 strikePrice, uint256 delta);
+
+    event PremiumDiscountSet(
```

Comment:
function associated with this event was removed

Reply:
Fixed in [a30c9d0](https://github.com/ribbon-finance/ribbon-v2/pull/297/commits/a30c9d05c95b9d301e7c16500455108d96450e59)